packages/health-check/src/utils.js

```diff
@@ -1,4 +1,7 @@
-const got = require("got");
+const ipCheckService = "whatismyip.akamai.com";
+const ipRegex = new RegExp(
```

Comment:
Should we care about IPv6 at this point?

Reply:
I don't think so, if we'll have to support ipv6 then it will mean some major refactor anyway I believe